presto-example-http/pom.xml

```diff
@@ -26,6 +26,13 @@
             <artifactId>json</artifactId>
         </dependency>
 
+        <dependency>
+            <!-- airlift json has a dependency on joda -->
+            <groupId>io.airlift</groupId>
+            <artifactId>joda-to-java-time-bridge</artifactId>
+            <scope>runtime</scope>
```

Comment:
will you remember to remove this when you make `joda-to-java-time-bridge` a SPI's dependency?